docs/release-notes/v11.3.0.md

```diff
@@ -39,9 +39,9 @@ For more information, [go to the documentation](/development_suite/governance/de
 
 ![Design Overview](./img/overview/design-overview-update.png)
 
-#### `⌘ + S` (`ctrl + S`) shortcut for saving configuration
+#### Shortcut for saving configuration process:`⌘ + S` (`ctrl + S`)
 
-It is possible to use `⌘ + K` (alternatively, `ctrl + K`) to speed-up your saving process, allowing to automatically open the Save configuration modal.
+It is possible to use `⌘ + S` (alternatively, `ctrl + S`) to speed-up your saving process, allowing to automatically open the Save configuration modal.
```

Comment:
let's put here a notice about being forced to save on another branch if someone has already saved on the same branch

Reply:
inserted a note about